assets/gui/start-screen.json

```diff
@@ -3,16 +3,34 @@
         {
             "name": "start-button",
             "path": "assets/gui/start-button.png"
+        },
+        {
+            "name": "start-selected-button",
+            "path": "assets/gui/start-selected-button.png"
+        },
+        {
+            "name": "load-button",
+            "path": "assets/gui/load-button.png"
         }
     ],
     "buttons": [
         {
-            "selected_image_asset": "start-button",
+            "selected_image_asset": "start-selected-button",
             "unselected_image_asset": "start-button",
-            "name": "start-button",
+            "name": "start-selected-button",
+            "down": "load-button",
             "center": [500, 325],
             "action": "game.gui.start.start_game"
+        },
+        {
+            "selected_image_asset": "load-button",
```

Comment:
We should also have a load-selected button asset so that when they move down to select the load button, it highlights it.